index.html

```diff
@@ -0,0 +1,2281 @@
+                      />
+                    </a>
+                  </div>
+                  <div class="info">
+                    <a
+                      href="/protesters-white-house-demand-trump-moscow-mitch-take-action-gun-violence-1452869"
+                      >Protesters Outside White House Demand 'Moscow Mitch' Act
+                      on Gun Violence</a
+                    >
+                  </div>
+                </li>
+              </ul>
+            </div>
+          </div>
+
+          <div class="sponsorship">
+            <div class="card">
+              <div class="wrapper">
+                <img
+                  class="card-img-top"
+                  src="https://d.newsweek.com/en/full/1511786/newsweek-momentum-awards-2019.png?w=397&h=265&q=90&f=b5bee11f8b144487ff291880f4d318be"
+                  alt="img"
+                />
+                <h4 class="card-cat card-group">ANALYSIS</h4>
+              </div>
+              <div class="card-info">
+                <h5 class="card-title">The Newsweek Momentum Awards</h5>
+                <p class="card-body p-0 text-justify">
+                  The Newsweek Momentum Awards will recognize five individuals
+                  around the globe, as well as one city, which will be chosen as
+                  the "smartest city in the world."
+                </p>
+              </div>
+            </div>
+          </div>
+
+          <div class="newsletter-signup jumbotron mt-3 mb-3">
+            <div class="n-title flex-xs ai-c">
+              <span class="icon-n"></span>Sign up for our Newsletter
+            </div>
+            <a href="https://email.newsweek.com/landing/newsletter" class="lk"
+              >Sign up</a
+            ><br />
+            <a href="https://email.newsweek.com/landing/newsletter" class="lk-5"
+              >Update your preferences »</a
+            >
+          </div>
+
+          <div class="sponsorship">
+            <div class="card">
+              <div class="wrapper">
+                <img
+                  class="card-img-top"
+                  src="https://d.newsweek.com/en/full/1511786/newsweek-momentum-awards-2019.png?w=397&h=265&q=90&f=b5bee11f8b144487ff291880f4d318be"
+                  alt="img"
+                />
+                <h4 class="card-cat card-group">ANALYSIS</h4>
+              </div>
+              <div class="card-info">
+                <h5 class="card-title">The Newsweek Momentum Awards</h5>
+                <p class="card-body p-0 text-justify">
+                  The Newsweek Momentum Awards will recognize five individuals
+                  around the globe, as well as one city, which will be chosen as
+                  the "smartest city in the world."
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+
+    <section class="container-fluid page-bottom">
```

Comment:
#### `HINT`

Consider using an heading (`h2`-`h6`) element in a `<section>` element.